### DISCUSSION TITLE
SmartAutoClicker-2.0.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#236] [Announcements]
Minor release fixing display issues on some devices.

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.0.1'>SmartAutoClicker-2.0.1</a>.</em>

**Comment by shmaungms** [comment]
Quick check looks good. Nothing hidden or cut off and the add new scenario appears and disappears consistently.

**Comment by TeH07** [comment]
The bug on my screen display 21:9 has been fixed.

**Comment by idxuantruong** [comment]
hello, thank you for sharing the source code of this application, i want to edit the source code of this application, what software should i use, for example android studio ....., thank you very much.

**Comment by Stanislav1710** [comment]
just a great app. you can do everything, the only thing lacking is the very function of clicking at a certain distance from the template found. Is it feasible and should we expect it in an update?
